doc/reference/bluetooth/hci.txt

```diff
@@ -42,7 +42,8 @@ command is supported.
 	|       | 4    | Set Scan Request Reports                   |
 	|       | 5    | Write Tx Power Level (per Role/Connection) |
 	|       | 6    | Read Tx Power Level (per Role/Connection)  |
-	|       | 7    | Reserved                                   |
+	|       | 7    | Read USB Mode                              |
+	|       | 8    | Set USB Mode                               |
```

Comment:
These are per octet blocks and Set USB Mode needs to go into octet 2.

Reply:
Right, I missed that.